src/types/api/billing.ts

```diff
@@ -0,0 +1,12 @@
+import { LoginStatistic } from '../../types'
+
+// yr: The year to get the billing information for
+// mth: The month to get the billing information for
+// esrvcId: The id of the form
+export type BillingQueryDto = {
+  esrvcId: string
+  yr: string
+  mth: string
+}
```

Comment:
is this really a DTO? it's only used internally by the frontend right?

Reply:
it's used by fe and be. the be uses it as part of `req.body`'s shape in the type of `RequestHandler`

Reply:
it's used in `src/app/modules/billing/billing.controller.ts`

Reply:
ahh I see that they are query params, got it